content/tutorial/tutorial.md

```diff
@@ -440,7 +440,7 @@ class Square extends React.Component {
     return (
       <button
         className="square"
-        onClick={() => this.props.onClick()}
+        onClick={() => this.props.onClick}
```

Comment:
That wouldn't call the `onClick` prop now though which is what we wanted.

Could elaborate how you reproduce the "Error: Maximum update depth exceeded" with this particular code?

Reply:
Sorry for not seeing this before. I was going through the tutorial from scratch. When I get to this part of the tutorial, I get the error. However changing it to `onClick` fixed the issue.

Reply:
> When I get to this part of the tutorial, I get the error.

Sorry, I'm not following what you mean by this. You you explain in detail what you did (from the start) to produce the error? Ideally include the full code that you produced e.g. in a [codesandbox](https://react.new)